scripts/ci-entrypoint.sh

```diff
@@ -141,6 +141,16 @@ wait_for_nodes() {
     "${KUBECTL}" get nodes -owide
 }
 
+copy_secret() {
+    unset KUBECONFIG
```

Comment:
could you please add comments to explain this is to point at the management cluster and that below we're setting the kubeconfig back to the workload cluster?

Reply:
> could you please add comments to explain this is to point at the management cluster and that below we're setting the kubeconfig back to the workload cluster?

Of course. Updated